en/docs/learn/consume-api/customizations/customizing-the-developer-portal/overriding-developer-portal-theme.md

```diff
@@ -12,32 +12,84 @@ Make sure to take a backup of the `defaultTheme.js` before making any changes.
 
 Changes done in the `defaultTheme.js` file are reflected directly in the Developer Portal ( It's not required to restart the server or rebuild the source code).
 
+!!!Note
+    API Manager Devportal is themed using React Material Design. The theme configuration is an external JSON file resides outside the React codebase. While an administrator who has access to the file system can override the default theme configuration, a tenant admin can override both of them via the defaultTheme.json file. The parameters you can override via the theme are listed at the bottom. Additionally, the default theme params available with React Material Design library can be overridden via this file. Following link list down the available params  [https://material-ui.com/customization/default-theme/#default-theme](https://material-ui.com/customization/default-theme/#default-theme)
+
```

Comment:
Additionally, we have introduced custom parameters to the theme file. You can refer to the following file for available parameters.
[defaultTheme.js](https://github.com/wso2/carbon-apimgt/blob/08ea54ec6be19ca6317027b09aaa566d6da85037/features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/devportal/source/src/defaultTheme.js)
Note that you need to only put the parameters that you override to your file.

Reply:
fixed with https://github.com/wso2/docs-apim/pull/879/commits/0dcd47129cb53f6aadfe9ab026bde349622a938e